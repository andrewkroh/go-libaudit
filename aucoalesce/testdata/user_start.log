type=USER_START msg=audit(1492571821.754:489): pid=3235 uid=0 auid=0 ses=7 msg='op=PAM:session_open acct="root" exe="/usr/sbin/cron" hostname=? addr=? terminal=cron res=success'