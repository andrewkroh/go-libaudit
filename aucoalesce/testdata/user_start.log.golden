{
  "@timestamp": "2017-04-19T03:17:01.754Z",
  "auid": "0",
  "category": "user-login",
  "result": "success",
  "sequence": 489,
  "session": "7",
  "user_start": {
    "acct": "root",
    "exe": "/usr/sbin/cron",
    "op": "PAM:session_open",
    "pid": "3235",
    "terminal": "cron",
    "uid": "0"
  }
}
